packages/storefront/packages/nuxt/__tests__/app/package.json

```diff
@@ -10,6 +10,6 @@
   },
   "devDependencies": {
     "nuxt": "3.7.4",
-    "@vue-storefront/sdk": "*"
+    "@vue-storefront/sdk": "1.4.0-rc.3"
```

Comment:
do we need to specify the exact version?

Reply:
not really just packages are not linked in monorepo and I needed to use it for test purposes.

Reply:
it's gonna be resolved after merging https://github.com/vuestorefront/vue-storefront/pull/7052